### ISSUE TITLE
`make dev` does not work

### ISSUE THREAD

**Issue by catilac** [tractordev/wanix#57]
I get this issue when I run `make dev` but I found a fix, PR coming
```
make dev
mkdir -p local/bin
cd kernel && GOOS=js GOARCH=wasm go build -ldflags="-X 'main.Version=0.1'" -o ../local/bin/kernel .
# tractor.dev/wanix/kernel/fs
fs/fs.go:73:9: s.fsys.MkdirAll undefined (type *watchfs.FS has no field or method MkdirAll)
make: *** [kernel] Error 1
```

**Comment by catilac** [comment]
https://github.com/tractordev/wanix/compare/main...catilac:wanix:fs_fix?expand=1

**Comment by catilac** [comment]
https://github.com/tractordev/wanix/pull/58

**Comment by Parzival-3141** [comment]
Closed by #58
